edit-post/store/actions.js

```diff
@@ -22,6 +22,30 @@ export function closeGeneralSidebar() {
 	};
 }
 
+/**
+ * Returns an action object used in signalling that the user opened an editor sidebar.
+ *
+ * @param {string} name        Sidebar name to be opened.
+ * @return {Object}            Action object.
```

Comment:
Indentation here is a bit off. I think all of the fields are supposed to line up, and `@return` should have a blank line above it.

(I know, I know, we really need lint rules for this...)
```
 * @param {string} name Sidebar name to be opened.

 * @return {Object} Action object.
```